APIKit/Session.swift

```diff
@@ -76,6 +68,36 @@ public class API {
             }.forEach { $0.cancel() }
         }
     }
+    
+    // Shared session for static methods
+    public static let sharedSession = Session(URLSession: NSURLSession(
+        configuration: NSURLSessionConfiguration.defaultSessionConfiguration(),
+        delegate: URLSessionDelegate(),
+        delegateQueue: nil
+    ))
+    
+    public static func sendRequest<T: RequestType>(request: T, handler: (Result<T.Response, APIError>) -> Void = {r in}) -> NSURLSessionDataTask? {
+        return sharedSession.sendRequest(request, handler: handler)
+    }
+    
+    public static func cancelRequest<T: RequestType>(requestType: T.Type, passingTest test: T -> Bool = { r in true }) {
+        sharedSession.cancelRequest(requestType, passingTest: test)
+    }
+}
+
+@available(*, deprecated, message="API is renamed as Session.")
```

Comment:
It might be better to make it unavailable for clarifying. I'll update this as `unavailable`, and leave message that indicates it is renamed.